Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj

```diff
@@ -73,7 +75,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Common\Common.csproj">
       <Project>{b3db0521-d9e3-4f48-9e2e-e5ecae886049}</Project>
-      <Name>Microsoft.PythonTools.Common</Name>
+      <Name>Common</Name>
```

Comment:
not sure why you shortened the name. previous one is a bit more clear

Reply:
This was necessary to get this to find it when building dependencies. The project reference had to match the project name.